lib/components/Button/index.tsx

```diff
@@ -103,12 +124,7 @@ export class ButtonImpl extends React.Component<ButtonProps, {}> {
   };
 
   render() {
-    const {
-      classes,
-      className: upperClassName,
-      btnSize, btnStyle,
-      sheet: _,
```

Comment:
deconstruct the prop `isLoading` as well, otherwise it will be passed to the `<button>` element via `buttonProps` and we'll end up with:
<img width="510" alt="screenshot 2018-10-27 12 03 30" src="https://user-images.githubusercontent.com/687961/47602508-53679880-d9e0-11e8-836b-071c034e61ad.png">